lib/_stream_duplex.js

```diff
@@ -61,6 +61,12 @@ function Duplex(options) {
   this.once('end', onend);
 }
 
+Object.defineProperty(Duplex.prototype, 'writeBuffer', {
+  get: function() {
+    return this._writableState.getBuffer();
```

Comment:
This needs to be protected, as `this._writableState`  could not be initialized yet.